### DISCUSSION TITLE
Error during  Dbeaver 21.2.0 setup to access the Z/os DB2 tables

### DISCUSSION THREAD

**Discussion by rajeevv2021** [dbeaver/dbeaver#14559] [General]
### Discussed in https://github.com/dbeaver/dbeaver/discussions/14343

<div type='discussions-op-text'>

<sup>Originally posted by **rajeevv2021** October 22, 2021</sup>
Hello -

I am getting the below error when I am setting up the Dbeaver 21.2.0 setup to access the Z/os DB2 tables .

Can't create driver instance
  Error creating driver 'DB2 z/OS' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'com.ibm.db2.jcc.DB2Driver'
  Error creating driver 'DB2 z/OS' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'com.ibm.db2.jcc.DB2Driver'
    com.ibm.db2.jcc.DB2Driver
    com.ibm.db2.jcc.DB2Driver

Kindly let me know how to get this fixed ?

Regards
Rajeev V</div>
![Capture1](https://user-images.githubusercontent.com/92970768/138402068-a9296a68-b0b4-4b63-a589-f46278216ca3.PNG)
![Driver Files Installed](https://user-images.githubusercontent.com/92970768/138402072-2e63911e-1678-46a1-ae0f-e823b626e5ad.PNG)

**Comment by uslss** [comment]
@rajeevv2021 seems something is going wrong with the connection. Probably you've got proxy enabled or firewall. Is it reproducible for all the drivers or for db2/zos only?
